tests/contrib/minikube/test_kubernetes_executor.py

```diff
@@ -65,8 +65,27 @@ def _get_session_with_retries(self):
         session.mount('https://', HTTPAdapter(max_retries=retries))
         return session
 
+    def _ensure_airflow_webserver_is_healthy(self):
+        http_codes = []
+        for _ in range(0, 10):
+            result = self.session.get(
+                "http://{host}/health".format(host=get_minikube_host()))
+            http_codes.append(result.status_code)
+            if result.status_code == 200:
+                break
+            time.sleep(4)
+
+        self.assertTrue(
+            200 in http_codes,
+            ("Airflow webserver is not healthy. "
+             "Got the following http codes: {}").format(http_codes)
+        )
+
     def setUp(self):
         self.session = self._get_session_with_retries()
+        output = check_output(['kubectl', 'get', 'pods']).decode()
+        print('Active k8s pods:\n{}'.format(output))
```

Comment:
This will produce a lot of "spam" in the logs and I don't think we should print this unless there is a problem.

Reply:
Sorry @ashb this still WIP. will remove it later